src/profile_service/index.js

```diff
@@ -17,7 +17,7 @@ module.exports.getReceipt = (token, pem, appId) => {
         try {
           const receipt = response.getReceipt();
           const parsedResponse = response.getParsedResponse();
-          const decryptedProfile = yotiCommon.decryptCurrentUserReceipt(receipt, pem);
+          const decryptedProfile = yotiCommon.decryptCurrentUserReceipt(receipt, pem, true);
```

Comment:
Why do we need this boolean?  Shouldn't we always just parse the full list of attributes?

Reply:
I agree, this wasn't the best approach - I've refactored in ecf8863 and updated the PR description, so it should be a little better now.  This needs to be refactored in the next major, which we have a ticket for.